build.sbt

```diff
@@ -652,9 +656,9 @@ lazy val bench = project
   .enablePlugins(JmhPlugin)
 
 lazy val binCompatTest = project
-  .disablePlugins(CoursierPlugin)
   .settings(noPublishSettings)
   .settings(
+    useCoursier := false,
```

Comment:
Coursier was disabled because it seemed to me that it doesn't understand this setting:
`dependsOn(core.jvm % Test)`

I should've added a comment. Can we add one now?